src/index.ts

```diff
@@ -95,26 +95,26 @@ class DebuggerHandler<H extends ConsoleHandler | NotebookHandler> {
     T extends IConsoleTracker | INotebookTracker,
     W extends ConsolePanel | NotebookPanel
   >(debug: IDebugger, tracker: T, widget: W): void {
-    if (!debug.model || this.handlers[widget.id]) {
-      return;
-    }
-    const handler = new this.builder({
-      debuggerService: debug,
-      widget
-    });
-    this.handlers[widget.id] = handler;
-    widget.disposed.connect(() => {
-      handler.dispose();
-      delete this.handlers[widget.id];
-    });
+    if (debug.model && !this.handlers[widget.id] && debug.isDebuggingEnabled) {
```

Comment:
This reverts the change from https://github.com/jupyterlab/debugger/pull/221/commits/226e55a9127533cd0f3a4f1a6eb4e09c0574d2a7.

Reply:
good catch. I missed that during make rebase